### PR TITLE
UI for transaction Actions

### PR DESCRIPTION
**Pull request by gguijarro-c-chwy**
More UI for Transaction actions

image

### CODE REVIEW

File: lib/screens/app/app.dart [hunk 1]
```diff
@@ -42,7 +42,9 @@ class _AppState extends State<App> with WidgetsBindingObserver {
       title: "Scan",
       icon: 'assets/images/scan.svg',
       iconSelected: 'assets/images/scan.svg',
-      screen: SendScannerScreen(),
+      screen: ScanConfirmationScreen(),
+      // TODO(gguij): remove this to show the scanner
```
Comment (gguijarro-c-chwy) [inline lib/screens/app/app.dart:46]: placeholder to show the view

File: lib/screens/transfer/scan/scan_confirmation_screen.dart [hunk 2]
```diff
@@ -6,6 +6,27 @@ import 'package:hashed/domain-shared/event_bus/event_bus.dart';
 import 'package:hashed/domain-shared/event_bus/events.dart';
 import 'package:hashed/domain-shared/page_command.dart';
 import 'package:hashed/screens/transfer/scan/interactor/viewmodels/scan_confirmation_bloc.dart';
+import 'package:hashed/screens/transfer/scan/scan_confirmation_action.dart';
+
+final mockData = [
```
Comment (gguijarro-c-chwy) [inline lib/screens/transfer/scan/scan_confirmation_screen.dart:11]: mocked data, This data should be passed by the scanner screen into this screen

**Comment by gguijarro-c-chwy** [comment]
image

**Comment by gguijarro-c-chwy** [comment]
image

Success

File: lib/screens/transfer/scan/scan_confirmation_screen.dart [hunk 3]
```diff
@@ -5,7 +5,32 @@ import 'package:hashed/design/lib/hashed_body_widget.dart';
...
+  ScanConfirmationAction(
+      data: ScanConfirmationActionData(actionName: const MapEntry('CreateRecovery', 'Recovery'), actionParams: {
+    'first': 'First Name',
+    'second': 'second name',
+    'third': 'Third =Nik',
+  })),
+  ScanConfirmationAction(
+      data: ScanConfirmationActionData(actionName: const MapEntry('CreateRecovery', 'Recovery'), actionParams: {
+    'Parameter 1': 'doe_john',
+    'Parameter 2': 'butt_roman12',
```
Comment (n13) [inline lib/screens/transfer/scan/scan_confirmation_screen.dart:25]: whats up with Jason and naming people "butt" 🤣

**Review by n13 — APPROVED** [review]
LGTM

**Pull request merged**